### DISCUSSION TITLE
How to update all numeroues modules

### DISCUSSION THREAD

**Discussion by JohanssonMarkus** [microsoftgraph/msgraph-sdk-powershell#1773] [Q&A]
There are quite a lot of modules to update each month. Hot to do this the best way? 
Shall I run update-module or uninstall/install-module? 
I guess a script is needed to do this (if not spending hours to make the update) 
Tried to create som basic script but it always failed in some part and takes lot of time to fix it every time. 

Any one do have some guidelines on how to update all graph modules (also beta) at once. 

Would be good if Microsoft would provide some Script or cmdlet to update them all at once.

**Comment by shivtorov** [comment]
Works fine for me:
`Update-Module -Name *`

**Comment by EricMooreHays** [comment]
Why does update-module always leave the prior version installed?
